third_party/maya/lib/usdMaya/usdTranslatorExport.mel

```diff
@@ -142,6 +142,11 @@ global proc int usdTranslatorExport (string $parent,
         checkBox -l "Export Visibility" exportVisibilityCheckBox;
 
         checkBox -l "Renderable Only" renderableOnlyCheckBox;
+
+        textFieldGrp -l "Filtered Node Types"
+            -annotation ("Do not filter nodes of this type, or their children;\n" +
```

Comment:
This annotation seems incorrect, should the description be something like "Filter out nodes of this type and their children" or "Do not import nodes of this type or their children"?